_config.yml

```diff
@@ -223,13 +244,34 @@ t:
     manifesto:
       title: "Who are we?"
       phrase1: "<b>Pyronear</b> is a non-profit NGO whose objective is to democratize open and low-tech solutions for fighting wildfires, for the benefit of the ecosystems and the citizens. To achieve this goal, we are co-building an early detection solution that is open source, efficient, automatic, energy-efficient, economical and modular"
-      phrase2: "<b>Open source</b>: All our code is available for free and co-developed by our volunteers. We believe that collective intelligence is the best way to solve a complex problem."
-      phrase3: "<b>Early detection</b>: To fight effectively against wildfires, it is important to detect them early in order to allow a quick intervention of fire and rescue services and thus limit their impact."
-      phrase4: "<b>Performance</b>: Our AI experts actively follow the news in this fast-moving field in order to make Pyronear benefit from its latest advances."
-      phrase5: "<b>Automatic</b>: We offer a fully automated solution to free up firefighters' time."
-      phrase6: "<b>Energetically sober</b>: We are in a so-called low-tech approach, we have built a low-consumption device allowing an easier deployment and a more reasoned use of our precious resources."
-      phrase7: "<b>Economic</b>: Our R&D is done by volunteers, our device is sober and the non-profit object of our association, which allows us to propose very competitive prices to facilitate a deployment on the territory."
-      phrase8: "<b>Modular</b>: We offer a complete detection system but our device has been designed to work with existing equipment (use of cameras already in place, connection with existing software ...)"
+      card1:
+        icon_src: "/img/cards_icons/opensource.svg"
+        title: "Open source"
+        content: "<b>Open source</b>: All our code is available for free and co-developed by our volunteers. We believe that collective intelligence is the best way to solve a complex problem."
```

Comment:
Typo, remove ```<b>Open source</b>``` in content as it is already in the "title"

Reply:
I did not translate, theses parts were already done in previous PR ! I did only add the icon_src and split the phrase into title and content.